src/travel-details-screens/DepartureDetailsScreenComponent.tsx

```diff
@@ -146,16 +148,27 @@ export const DepartureDetailsScreenComponent = ({
                         : DepartureDetailsTexts.map,
                     )}
                     interactiveColor="interactive_1"
-                    onPress={() =>
+                    onPress={() => {
+                      vehiclePosition &&
+                        analytics.logEvent(
+                          'Trip search',
```

Comment:
Not sure how this works, but where is this context/keyword defined? Sounds like we maybe should have one for departure details.

Reply:
I'm not sure what the difference between 'Trip details' and 'Departure details' is, but the contexts are defined in `src/analytics/analytics-event-context.ts`. I don't have an strong opinions on what these contexts should be

Reply:
It would be nice to use `Departure details` when the user enters the live view from the departure details. If you want to view a departure you go to departure details. Then possibly to live busses. Trip details is from viewing a specific trip suggestion.

How to handle the definitions is up to you.